site/src/pages/CliAuthPage/CliAuthPageView.tsx

```diff
@@ -21,11 +24,14 @@ export const CliAuthPageView: FC<CliAuthPageViewProps> = ({ sessionToken }) => {
       <Welcome>Session token</Welcome>
 
       <p css={styles.instructions}>
-        Copy the session token below and{" "}
-        <strong css={{ whiteSpace: "nowrap" }}>
-          paste it in your terminal
-        </strong>
-        .
+        Copy the session token below and
+        {/*
+         * This looks silly, but it's a case where you want to hide the space
+         * visually because it messes up the centering, but you want the space
+         * to still be available to screen readers
+         */}
+        <span css={{ ...visuallyHidden }}>{VISUALLY_HIDDEN_SPACE}</span>
+        <strong css={{ display: "block" }}>paste it in your terminal.</strong>
```

Comment:
Changed the previous markup because the `nowrap` styling had a risk of breaking the text rendering at smaller viewports